bin/download-syntax

```diff
@@ -280,6 +280,15 @@ REPOS = (
         license_path='LICENSE.txt',
         grammars=('syntaxes/SQL.plist',),
     ),
+    Repo(
+        name='microsoft/vscode-textmate',
+        version='975e2e3',
+        license_path='LICENSE.md',
+        grammars=(
+            'test-cases/themes/syntaxes/coffeescript.json',
+            'test-cases/themes/syntaxes/Regular Expressions (JavaScript).tmLanguage',  # noqa: E501
```

Comment:
hmmm I don't think these are real grammars since they're pulled out of the tests of the textmate engine

Reply:
the test file had to have come from _somewhere_ though -- I'm not really comfortable using a file from tests

Reply:
There is a grammar in [microsoft/vscode](https://github.com/microsoft/vscode/blob/main/extensions/coffeescript/syntaxes/coffeescript.tmLanguage.json), though the base for this is an atom project which is dead. Another alternative could be forking and updating (textmate/coffee-script.tmbundle)[https://github.com/textmate/coffee-script.tmbundle] as this project is dead too and doesn't seem to provide as much rules as the vscode one.

Reply:
we pull some other ones directly from vscode -- this one is probably fine too